raiden/api/python.py

```diff
@@ -1740,7 +1766,7 @@ def create_light_client_payment(
                                                     message=locked_transfer, is_signed=False)
             return HubResponseMessage(lcpm_id, LightClientProtocolMessageType.PaymentSuccessful, payment_hub_message)
         else:
-            raise ChannelNotFound("Channel with given partner address doesnt exists")
+            raise ChannelNotFound("Your light client has 0 channels oppened")
```

Comment:
This is not true after this routing modification.

Node A channels: B,C,D,E. Node E and D have a channel with F.

Node A sends a payment to node F, the first try to get a direct channel will fail, the routing one too (because there is no balance to route from A to F i.e). Therefore channel_state will be none, but the light client has B,C,D,E channels, not zero.